nxapi/nxlog.py

```diff
@@ -34,9 +64,28 @@ def parse_nxlog(nxlog):
         return errors, ret
 
     # Flatten the dict, since parse_qs is a bit annoying
-    ret = parse_qs(nxlog[start:end])
-    for key, value in ret.items():
-        ret[key] = value[0]
+    raw_dict = parse_qs(nxlog[start:end])
+    for key, value in raw_dict.items():
+        raw_dict[key] = value[0]
+
+    # We may have a multi-line event
+    min_dict = dict()
+    for key in raw_dict:
+        if not key.startswith('id') and not key.startswith('zone') and not key.startswith('var_name'):
+            min_dict[key]=raw_dict[key]
+
+    min_dict['date'] = unify_date(date)
+    min_dict['coords'] = coords(min_dict['ip'])
+    
+    for i in itertools.count():
+        if "id%d" % i in raw_dict and "var_name%d" %i in raw_dict and "zone%d" %i in raw_dict:
```

Comment:
What about something like:
```python
_id = "id%s" % i
var_name = "var_name%d" % i
raw_dict = "raw_dict%d" % i
```
to reuse variables instead of having `"bla%d" % i` all around?